psydac/api/tests/test_api_2d_fields.py

```diff
@@ -320,14 +326,15 @@ def test_field_collela():
     assert abs(error_2 - expected_error_2) < 1.e-7
 
 #------------------------------------------------------------------------------
-def test_field_quarter_annulus():
+@pytest.mark.parametrize('do_plot', [False, True])
+def test_field_quarter_annulus(do_plot):
```

Comment:
If you parametrize the test, `pytest` will show the plot when the unit tests are run. Is this what you wanted? If not, I suggest simply
```suggestion
def test_field_quarter_annulus(do_plot=False):
```
which runs the unit test with `do_plot=False`, but allows you to run it manually with `do_plot=True`.

Reply:
Well the point was to test the plot. 
As the argument `hide_plot=True` is passed to `plot_field`, I imagine the plot will be saved to file but not shown.

Reply:
I guess you're right! I will put this in a separate PR

Reply:
(I don't really see how to test the content of the plots)